### PR TITLE
Tags for volumes in an Instance

### PR DESCRIPTION
**Pull request by nvnyale**

### CODE REVIEW

File: ec2/tags.go [hunk 1]
```diff
@@ -19,6 +20,43 @@ func (e *Ec2) UpdateTags(ctx context.Context, rawTags map[string]string, ids ...
...
 	}
 
+	instanceIDs := []*string{}
+	for _, id := range ids {
+		if strings.HasPrefix(id, "i-") {
+			instanceIDs = append(instanceIDs, aws.String(id))
+		}
+	}
+
+	describeVolumesInput := ec2.DescribeVolumesInput{
```
Comment (tenyo) [inline ec2/tags.go:30]: I don't think all this logic belongs here - should probably move to an orchestrator function.

Comment (tenyo) [reply ec2/tags.go:30]: We already have a `ListInstanceVolumes` function that does this, so why not just call that?

**Review by tenyo — CHANGES_REQUESTED** [review]

**Review by tenyo — COMMENTED** [review]
Excellent, this is much better! Let's confirm that it works as expected and we can merge it

File: ec2/tags.go [hunk 2]
```diff
@@ -70,3 +32,17 @@ func (e *Ec2) UpdateTags(ctx context.Context, rawTags map[string]string, ids ...
 
 	return nil
 }
+
+func (e *Ec2) UpdateInstanceTags(ctx context.Context, input *ec2.CreateTagsInput) error {
```
[the remark this answers is not in the document]
Comment (nvnyale) [reply ec2/tags.go]: Renamed.

**Review by tenyo — APPROVED** [review]

**Pull request merged**